spec/lucky_cli/generators/validators_spec.cr

```diff
@@ -0,0 +1,40 @@
+require "../../spec_helper"
+
+private def validate(string)
+  LuckyCli::Generators::Validators::ProjectName.valid? string
+end
+
+private def sanitize(string)
+  LuckyCli::Generators::Validators::ProjectName.sanitize string
+end
+
+
+describe LuckyCli::Generators::Validators do
+  context "ProjectName" do
```

Comment:
Rather than adding a context, I think this could be moved into the describe. That way it is a bit more inline with the rest of the specs. What do you think?